js/src/ui/StatusIndicator/statusIndicator.js

```diff
@@ -37,6 +37,11 @@ export default class StatusIndicator extends Component {
 
   render () {
     const { id, status, title, type, tooltipPlacement } = this.props;
+    const tooltip = title.find(x => !x.isEmpty) ? (
```

Comment:
Oh, I didn't notice the `data-tip`. Yes, in the case of health success we return an empty message. Alternatively, we could update the `HealthInfo` message to be an `Option`.

Yes, I agree it's probably not a good idea to use `isEmpty`. Maybe we can use lodash's `_.isEmpty`, or something like that which handles all the corner cases.